manifests/haproxy.yaml.in

```diff
@@ -44,5 +42,3 @@ spec:
           periodSeconds: 20
       securityContext:
         runAsNonRoot: true
-      nodeSelector:
```

Comment:
I was under the impression something about our haproxy multiplexing between the k8s api server and virt-api required special node placement. 

From looking at this in more detail, the haproxy config appears to be referencing DNS names instead of hard coded IPs, so it seems like the node selector wouldn't be necessary. 

Does removing the node select actually work with ci?

Reply:
Seems so, see result below.